scripts/velox_env_linux.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: velox_base
+
+channels:
+  - conda-forge
+
+variables:
+  CC: clang
+  CXX: clang++
+
+dependencies:
+# tools
+  - binutils
+  - bison
+  - clangxx
+  - cmake
+  - ccache
+  - flex
+  - gxx=12 # has to be installed to get clang to work...
```

Comment:
Is clang needed? I was incidentally trying to install velox in a conda env today, and I was writing my environment.yml file with the generic `compilers` dependency (which gives me gcc and not clang, and that seems to be ok)

Reply:
GCC also works but we are looking at implementing sanitizer checks which (due to folly internals iiuc) will be much easier when using clang.

Reply:
Seconded on why we need clang over gcc.

Reply:
I am happy to use gcc only, I added clang to facilitate your work on the *SAN checks.